packages/eds-core-react/src/components/Tooltip/Tooltip.docs.mdx

```diff
@@ -0,0 +1,62 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Tooltip } from '.'
+
+# Tooltip
+
+Shows when hovered, focused or long-pressed (touch), and display information such as a text label or a short description.
```

Comment:
I am not sure if this claim about long-press is true, in our implementation at least? (on my android phone it activates on tap, and i think we need to implement long-press detection with a timeout or something if we want this)

Reply:
I will simplify the description.